docs/developer/architecture/alerting.rst

```diff
@@ -0,0 +1,37 @@
+Alerting Functionalities
+========================
+
+Context
+-------
+
+MetalK8s is automatically deploying Prometheus, Alertmanager and a set of
```

Comment:
Still need to create manually volumes

Reply:
yes, true, but no need to say it here in this requirement doc I think